lib/blockstore/index.js

```diff
@@ -0,0 +1,42 @@
+/*!
+ * blockstore/index.js - bitcoin blockstore for bcoin
+ * Copyright (c) 2019, Braydon Fuller (MIT License).
+ * https://github.com/bcoin-org/bcoin
+ */
+
+'use strict';
+
+const {join} = require('path');
+
+const AbstractBlockStore = require('./abstract');
+const LevelBlockStore = require('./level');
+const FileBlockStore = require('./file');
+
+/**
+ * @module blockstore
+ */
+
+exports.create = (options) => {
+  const location = join(options.prefix, 'blocks');
+
+  if (options.memory) {
+    return new LevelBlockStore({
+      network: options.network,
+      logger: options.logger,
+      location: location,
+      cacheSize: options.cacheSize,
+      memory: options.memory
+    });
+  }
+
+  return new FileBlockStore({
```

Comment:
Same comment as above: any reason to disallow other LevelDB configuration?
https://github.com/bcoin-org/bcoin/pull/703/files#r274256067

Reply:
Same as above.

Reply:
Other options can be added as needed, they will need to have a new config name as well.